### DISCUSSION TITLE
How to type the mock of groupBy ?

### DISCUSSION THREAD

**Discussion by Suniron** [prisma/prisma#19510] [Q&A]
Hey,

I can mock my groupBy calls but the typing is not working:

This code in test `prismaMock.dashboardItem.groupBy.mockResolvedValue(mockFetchedBreakpoints)` will display this error:

```ts
Property 'mockResolvedValue' does not exist on type '{} & (<T extends DashboardItemGroupByArgs, HasSelectOrTake extends Or<"skip" extends Keys<T> ? 1 : 0, "take" extends Keys<T> ? 1 : 0>, OrderByArg extends 1 extends HasSelectOrTake ? { ...; } : { ...; }, OrderFields extends ExcludeUnderscoreKeys<...>, ByFields extends _TupleToUnion<...>, ByValid extends Has<...>, Hav...'.ts(2339)
```

Do you have an example where groupBy mock typing works?

**Comment by nurul3101** [comment]
Hello @Suniron 👋 

I was able to find this related Stack Overflow answer:
https://stackoverflow.com/questions/76083136/jest-mock-output-for-prisma-groupby-method

Is your use case similar to this?

**Reply by nurul3101** [reply]
@Suniron I am checking to see if you had a chance to look at my last response.
Please let us know in case you need any help.

**Reply by nurul3101** [reply]
I'll mark this discussion as closed. Please feel free to reopen if needed.
